src/supabase/profiles.ts

```diff
@@ -134,20 +137,23 @@ export const addToProfileWatchQueue = async (
 };
 
 /**
- * Remove a single show from a users watch queue
+ * Remove a single show from a users watch_queue, watched_queue, or favorites
```

Comment:
nit: This comment and the one below could be updated to watched, queue, favorites.